clash-prelude/src/Clash/Tutorial.hs

```diff
@@ -223,11 +223,20 @@ For installation instructions, see <https://clash-lang.org/install/ clash-lang.o
 
 {- $working
 This tutorial can be followed best whilst having the Clash interpreter running
-at the same time. If you followed the installation instructions, you already
-know how to start the Clash compiler in interpretive mode:
+at the same time. If you followed the installation instructions based on
+<https://docs.haskellstack.org/en/stable/README/#how-to-install Stack>, you can
+start the Clash compiler in interpretive mode by:
 
 @
-clash.clashi  # When installed from source, use @clashi@
+stack exec --package clash-ghc --resolver=nightly -- clashi
```

Comment:
In case you missed it:

https://github.com/clash-lang/clash-lang.org/pull/37#issuecomment-808888730

I don't think we need `--resolver=nightly`.

Reply:
Fixed